generic_config_updater/gu_common.py

```diff
@@ -162,6 +166,32 @@ def validate_field_operation(self, old_config, target_config):
                 if any(op['op'] == operation and field == op['path'] for op in patch):
                     raise IllegalPatchOperationError("Given patch operation is invalid. Operation: {} is illegal on field: {}".format(operation, field))
 
+        def _invoke_validating_function(cmd):
+            # cmd is in the format as <package/module name>.<method name>
+            method_name = cmd.split(".")[-1]
+            module_name = ".".join(cmd.split(".")[0:-1])
+            module = importlib.import_module(module_name, package=None)
+            method_to_call = getattr(module, method_name)
+            return method_to_call()
+
+        if os.path.exists(GCU_TABLE_MOD_CONF_FILE):
+            with open(GCU_TABLE_MOD_CONF_FILE, "r") as s:
+                gcu_table_modification_conf = json.load(s)
+        else:
+            raise GenericConfigUpdaterError("GCU table modification validators config file not found") 
+
+        for element in patch:
+            path = element["path"]
+            table = re.search(r'\/([^\/]+)(\/|$)', path).group(1) # This matches the table name in the path, eg. PFC_WD without the forward slashes
```

Comment:
># This matches the table name in the path, eg. PFC_WD without the forward slashes [](http://example.com/codeflow?start=66&length=81)

Regex is hard to read. Would you provide more code comment, like sample match case.

Reply:
yes, I added a sample match case in the comment